src/apps/statements/tests/getStatements/utils/assertStatementsPageResultsAndOrder.ts

```diff
@@ -0,0 +1,60 @@
+import assert from 'assert';
+import { isArray, isString } from 'lodash';
+
+import ClientModel from '../../../models/ClientModel';
+import Service from '../../../serviceFactory/Service';
+
+/* istanbul ignore next */
+export const assertStatementsPageResultsAndOrder = async (
+  {
+    service,
+    client,
+    expectedPageStatementIds,
+    limit = 2,
+    cursor,
+    isNextPageCheckEnabled = true,
+    ascending = false,
+    /* Should start from 1 */
+    pageNumber= 0,
+  }: {
+    readonly service: Service;
+    readonly client: ClientModel;
+    readonly expectedPageStatementIds: string[];
+    readonly limit?: number;
+    readonly isNextPageCheckEnabled?: boolean;
+    readonly ascending?: boolean;
+    readonly pageNumber?: number;
+    readonly cursor?: string;
+  },
+) => {
```

Comment:
Nice util, I think in future we should use the `opts` style object parameter used in [the Engineering Guide under the Too Many Function Parameters section](https://docs.google.com/document/d/13QdnXlv2_-WKHp5-YGEAHtXATlASc2fcA544DMCnSh8/edit#heading=h.f1e6ski1ys7). Ok with this in this PR though 👍